src/bin/rav1e.rs

```diff
@@ -206,22 +209,32 @@ fn do_encode<T: Pixel, D: Decoder>(
     y4m_enc.as_mut(),
   )? {
     for frame in frame_info {
-      progress.add_frame(frame);
       if verbose {
-        eprintln!("{} - {}", frame, progress);
+        let frame_text = frame.to_string();
+        progress.add_frame(frame);
+        info!("{} - {}", frame_text, progress);
       } else {
+        progress.add_frame(frame);
         eprint!("\r{}                    ", progress);
       };
     }
 
     output.flush().unwrap();
   }
-  eprint!("\n{}\n", progress.print_summary());
+  if !verbose {
+    eprint!("\r");
```

Comment:
This should probably not be here

Reply:
This is used to clear out the progress indicator at the end of encoding (in non-verbose mode). In verbose mode there is no progress indicator, we print every frame on a new line, so we don't need to clear anything on verbose mode.